src/jit/codegenarm.cpp

```diff
@@ -541,7 +511,7 @@ void CodeGen::genLclHeap(GenTree* tree)
         noway_assert(compiler->lvaReturnEspCheck != 0xCCCCCCCC &&
                      compiler->lvaTable[compiler->lvaReturnEspCheck].lvDoNotEnregister &&
                      compiler->lvaTable[compiler->lvaReturnEspCheck].lvOnFrame);
-        getEmitter()->emitIns_S_R(ins_Store(TYP_I_IMPL), EA_PTRSIZE, regCnt, compiler->lvaReturnEspCheck, 0);
+        getEmitter()->emitIns_S_R(ins_Store(TYP_I_IMPL), EA_PTRSIZE, REG_SPBASE, compiler->lvaReturnEspCheck, 0);
```

Comment:
This used to use regCnt, now it uses REG_SPBASE.
A comment about the instruction being generated here would be nice.
Is it still necessary to compute 'regCnt' above?

Reply:
Thanks for the offline explaination.